julearn/api.py

```diff
@@ -217,6 +215,7 @@ def run_cross_validation(
 def create_pipeline(
     model,
     confounds=None,
+    categorical_features=None,
```

Comment:
Why do we need this here? The pipeline should not care about the input data. This is something that is dealt with in the `fit` method.

Reply:
perhaps to auto hot-encode them?

Reply:
We did not yet add auto hot-encoding. 
But this should also be able as a feature for `run_cross_validation` if we put `categorical_features` into `fit_params`. 
I am not sure why we did not do that before, but I think we were not completely sure how to do it best and then I followed what we already had. 
I will try to implement both `categorical_featrues` and `confounds` into `fit_params` if possible in #142 .